package.json

```diff
@@ -1,33 +1,19 @@
 {
```

Comment:
Given the changes also affect this file, this PR may be a tad too aggressive.. Let's discuss on what we really need to preserve and go from there